pkg/loki/loki.go

```diff
@@ -55,6 +56,7 @@ type Config struct {
 	TableManager     chunk.TableManagerConfig    `yaml:"table_manager,omitempty"`
 	Worker           frontend.WorkerConfig       `yaml:"frontend_worker,omitempty"`
 	Frontend         frontend.Config             `yaml:"frontend,omitempty"`
+	TailProxy        tailproxy.Config            `yaml:"tail_proxy,omitempty"`
```

Comment:
In addition to what @sandeepsukhani  mentioned, I think this should be part of the frontend config rather than it's own top level value because it's only used in the frontend.

We may need to extend our frontend definition to be something like
```go
type Config struct {
    frontend.Config `yaml:",inline"`
    tail_proxy_url string `yaml:"tail_proxy_url"`
}
```

Reply:
That is a good idea. I did it like this because I knew it does not belong to Cortex.

This is better I will prepare it

Reply:
edit: i love the podracer gif :)

Reply:
@owen-d what do think about it now? I used embedded struct as you proposed and IMHO it is better.